app/js/lib/foxbox/foxbox.js

```diff
@@ -87,15 +61,10 @@ export default class Foxbox extends Service {
       .then(() => this[p.net].init())
       .then(() => {
         // Start polling.
-        this[p.settings].on('pollingEnabled', () => {
-          this.togglePolling(this[p.settings].pollingEnabled);
-        });
-        this.togglePolling(this[p.settings].pollingEnabled);
-
-        this.recipes = new Recipes({
-          settings: this[p.settings],
-          net: this[p.net],
+        this[p.settings].on('polling-setting-changed', () => {
```

Comment:
I left it here as it was to reduce affected surface for this PR is already huge :) But in the future I was thinking that services itself can trigger that...